src/components/ColumnHeader/ColumnHeader.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { cellStyle } from '../../helpers/cellStyle.js'
+import { Direction } from '../../helpers/sort.js'
+import { useLocalStorageState } from '../../hooks/useLocalStorageState.js'
+import ColumnResizer from '../ColumnResizer/ColumnResizer.js'
+import { measureWidth } from './ColumnHeader.helpers.js'
+
+interface Props {
+  children?: React.ReactNode
+  dataReady?: boolean
+  direction?: Direction
+  localStorageKey?: string
+  onClick?: (e: React.MouseEvent) => void
+  title?: string
+}
+
+export default function ColumnHeader({ children, dataReady, direction, localStorageKey, onClick, title }: Props) {
+  const ref = React.useRef<HTMLTableCellElement>(null)
+  const [width, setWidth] = useLocalStorageState<number>({ key: localStorageKey ? `${localStorageKey}:width` : undefined })
+  const [resizeWidth, setResizeWidth] = React.useState<number | undefined>(undefined)
+  const currentWidth = resizeWidth ?? width
+
+  const style = React.useMemo(() => {
+    return cellStyle(currentWidth)
+  }, [currentWidth])
+
+  // Measure default column width when data is ready
+  React.useEffect(() => {
```

Comment:
nit: i prefer importing from react instead of `React.`

Reply:
done.